crates/experimentation_platform/tests/experimentation_tests.rs

```diff
@@ -25,21 +25,24 @@ fn single_dimension_ctx_gen(value: Dimensions) -> serde_json::Value {
                 {"var": "clientId"}
             ]
         }),
-    }
+    };
+    Some(context)
 }
 
-fn multiple_dimension_ctx_gen(values: Vec<Dimensions>) -> serde_json::Value {
+fn multiple_dimension_ctx_gen(values: Vec<Dimensions>) -> Option<serde_json::Value> {
     let mut conditions: Vec<serde_json::Value> = vec![];
     for val in values {
-        conditions.push(single_dimension_ctx_gen(val));
+        if let Some(context) = single_dimension_ctx_gen(val) {
```

Comment:
If you see here you are doing `if let`. But if you just return `serde_json::Value` from `single_dimension_ctx_gen` you don't need to do this.

Reply:
Hey actually at other places we want the Option Value only like at test_are_overlapping_contexts function
We are using the Option <Value>